### ISSUE TITLE
Installation Tutorial for tar.gz file from bitcoin.org 

### ISSUE THREAD

**Issue by alexstrbonn** [bitcoin-dot-org/Bitcoin.org#3412]
There are many discussions going on as there is a lack of a simple tutorial as to how to install Bitcoin Core (GUI) on Linux distributions, i.e Ubuntu 18.04.

It would be great if Bitcoin.org would provide such instruction to reach out to a broader user base.
Not all users wish to use the Snapstore.

See also here: https://www.reddit.com/r/Bitcoin/comments/hka0qc/why_are_there_no_packages_for_bitcoin_in_linux/

Is there anybody who could write up a tutorial or just post a video for a safe & quick installation of the tar.gz file which is downloaded from Bitcoin.org?

**Comment by jdabs** [comment]
@alexstrbonn do you mean a doc about how to "build from source"?

**Comment by alexstrbonn** [comment]
> @alexstrbonn do you mean a doc about how to "build from source"?

Thanks, but no. I did not make myself clear:

Bitcoin.org offers the tar.gz file (i.e. available here: https://bitcoin.org/bin/bitcoin-core-0.20.0/bitcoin-0.20.0-x86_64-linux-gnu.tar.gz) which can be installed in Linux, i.e. Ubuntu Desktop 18.04.

However, I share the frustration of many users:

Bitcoin.org does not offer a "step-by-step" instruction to  on how to to install that file in a simple way. 
(Many users do not wish to use the snapstore for various reasons)

That would be very helpful for many and would attract a broader user base in Linux/Ubuntu.

Thx

**Comment by Cobra-Bitcoin** [comment]
Good idea, I'll write up the tutorial when I get some free time, I don't think very many users install it this way but it makes sense to at least have a guide for it.

**Comment by alexstrbonn** [comment]
@Cobra-Bitcoin 
There is "kind of" a tutorial here. I believe it can be used to guide linux users through the installation. Some similar steps could be posted on Bitcoin.org. It is not that difficult.
 
https://bitcoin.stackexchange.com/questions/96973/installation-instrucion-the-for-bitcoin-core-tar-gz-file-from-bitcoin-org-in-li

**Comment by lodembeep** [comment]
Seems like it is an old issue, just in case : https://bitcoin.org/en/full-node#linux-instructions

**Comment by lodembeep** [comment]
I checked the files, downloaded from bitcoin.org the tar.gz file, used tar -xf, the binaries are precompiled so there shouldn't be any problem.

The issue should be more about the required dependencies to launch the GUI, if you don't have the qt libs, it won't work.
